assets/config/config.yaml

```diff
@@ -13,6 +13,6 @@ operator:
   prometheusVersion: "v3.5.0" # Tracks scylla-monitoring/versions.sh PROMETHEUS_VERSION
 operatorTests:
   scyllaDBVersions:
-    updateFrom: "2025.1.4" # One patch lower than .operator.scyllaDBVersion
-    upgradeFrom: "6.2.3" # One minor lower than .operator.scyllaDBVersion
+    updateFrom: "2025.2.0" # One patch lower than .operator.scyllaDBVersion
```

Comment:
Shouldn't we use an rc here? Since we don't support 2025.2.0 (even if it's just due to the utils)